test/Ocean/SplitExplicit/test_vertical_integral_model.jl

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env julia --project
+using Test
+using ClimateMachine
+ClimateMachine.init()
+using ClimateMachine.GenericCallbacks
+using ClimateMachine.ODESolvers
+using ClimateMachine.Mesh.Filters
+using ClimateMachine.VariableTemplates
+using ClimateMachine.Mesh.Grids: polynomialorder
+using ClimateMachine.Ocean.HydrostaticBoussinesq
+using ClimateMachine.Ocean.ShallowWater
+using ClimateMachine.Ocean.SplitExplicit: VerticalIntegralModel
+
+using ClimateMachine.Mesh.Topologies
+using ClimateMachine.Mesh.Grids
+using ClimateMachine.BalanceLaws
+using ClimateMachine.BalanceLaws: number_state_auxiliary
+using ClimateMachine.DGMethods
+using ClimateMachine.DGMethods.NumericalFluxes
+using ClimateMachine.MPIStateArrays
+using ClimateMachine.VTK
+
+using MPI
+using LinearAlgebra
+using StaticArrays
+using Logging, Printf, Dates
+
+import ClimateMachine.Ocean.ShallowWater: shallow_init_state!, shallow_init_aux!
+
+using CLIMAParameters
+using CLIMAParameters.Planet: grav
+struct EarthParameterSet <: AbstractEarthParameterSet end
+const param_set = EarthParameterSet()
+
+struct GyreInABox{T} <: ShallowWaterProblem
+    τₒ::T
+    fₒ::T # value includes τₒ, g, and ρ
+    β::T
+    Lˣ::T
+    Lʸ::T
+    H::T
+end
+
+function shallow_init_state!(
+    m::ShallowWaterModel,
+    p::GyreInABox,
+    Q,
+    A,
+    coords,
+    t,
+)
+    @inbounds x = coords[1]
+
+    Lˣ = p.Lˣ
+    H = p.H
+
+    kˣ = 2π / Lˣ
+    νʰ = m.turbulence.ν
+
+    M = @SMatrix [-νʰ * kˣ^2 grav(m.param_set) * H * kˣ; -kˣ 0]
+    A = exp(M * t) * @SVector [1, 1]
+
+    U = A[1] * sin(kˣ * x)
+
+    Q.U = @SVector [U, -0]
+    Q.η = A[2] * cos(kˣ * x)
+
+    return nothing
+end
+
+function shallow_init_aux!(p::GyreInABox, aux, geom)
+    @inbounds y = geom.coord[2]
+
+    Lʸ = p.Lʸ
+    τₒ = p.τₒ
+    fₒ = p.fₒ
+    β = p.β
+
+    aux.τ = @SVector [-τₒ * cos(π * y / Lʸ), 0]
+    aux.f = fₒ + β * (y - Lʸ / 2)
+
+    return nothing
+end
+
+function test_vertical_integral_model()
+    mpicomm = MPI.COMM_WORLD
+    ArrayType = ClimateMachine.array_type()
+
+    ll = uppercase(get(ENV, "JULIA_LOG_LEVEL", "INFO"))
+    loglevel = ll == "DEBUG" ? Logging.Debug :
+        ll == "WARN" ? Logging.Warn :
+        ll == "ERROR" ? Logging.Error : Logging.Info
+    logger_stream = MPI.Comm_rank(mpicomm) == 0 ? stderr : devnull
+    global_logger(ConsoleLogger(logger_stream, loglevel))
```

Comment:
I don't think this is needed anymore, it's done in [`Driver.jl`](https://github.com/CliMA/ClimateMachine.jl/blob/1d183062da2687e4e3255a0302c22e21df8c1020/src/Driver/Driver.jl#L458-L470)

```suggestion
```

Reply:
Does this apply even if we aren't using driver? Is it still handled in `Clima.init()`?

I don't want to add the split-explicit setup to the Driver framework until we know what we actually need to do

Reply:
> Does this apply even if we aren't using driver? Is it still handled in `Clima.init()`?

I believe so. @kpamnany?

Reply:
[Yes](https://github.com/CliMA/ClimateMachine.jl/blob/master/src/Driver/Driver.jl#L457-L470).

Reply:
okay great, I'll remove them